src/configs/python.rs

```diff
@@ -22,7 +22,7 @@ impl<'a> RootModuleConfig<'a> for PythonConfig<'a> {
             pyenv_version_name: false,
             pyenv_prefix: "pyenv ",
             python_binary: VecOr(vec!["python", "python3", "python2"]),
-            format: "via [${symbol}${pyenv_prefix}(${version} )(\\($virtualenv\\))]($style)",
+            format: "via [${symbol}${pyenv_prefix}(${pyenv_version} )(v${version} )(\\($virtualenv\\))]($style)",
```

Comment:
For python maybe this would be a good time to make a bigger change and deprecate the `pyenv_prefix` and `pyenv_version_name` and just use the format string with the new `pyenv_version` variable. This would simplify the code and make the configuration a bit more consistent. This would be a breaking change at the moment to use the pyenv version you would have to set the following
```toml
[python]
pyenv_version_name = true
```
after this you would need to do
```toml
[python]
format = 'via [${symbol}pyenv ${pyenv_version} (\($virtualenv\))]($style)'
```
What do you think?

Reply:
I'm fine with with removing the `pyenv_prefix` since I don't see any benefits to it vs. inserting its value directly in the format string since I'm always editing the format string anyway, though there might be users who are fine with the format and only need to change a prefix? No clue how common it is
Re. the `pyenv_version_name` — if it's removed would this mean that every prompt refresh would need to make calls to both python **and** pyenv to update even if the user doesn't have/use pyenv? That might be a performance issue?

Otherwise I've incorporated all the other code simplifications and will push shortly to see how this affects all those CI checks

Reply:
The python and pyenv commands are lazily evaluated, so they would only be called when the `$version` and/or `$pyenv_version` variables are used in the format string.

Reply:
Ok, but the default format string does include both of these variables in the format string, so this doesn't solve this issue. If it is an issue, that is
Then this gets us back to my first point — how useful this extra bool var is for other users:
- is it that they simply edit the `format` string in their configs (like I do since I also need to get rid of all those useless `via `) and then it's largely irrelevant since you'd just as well remove the `$pyenv_version` if you don't need it
- or is it that many don't touch the `format` string, but only add `symbol`/`pyenv_version_name` vars

Reply:
Ah sorry, I didn't explain that side of things. I would just change the default format string to add the `v` so that it is  something like `via [${symbol}(v${version} )(\($virtualenv\) )]($style)'`. So if a user wants to use the pyenv version they would just modify the format string.

Reply:
Oh, yeah, that's fine, I'll try to change this later
Would it be better, maybe, to make it a separate PR given it's a breaking change?

Reply:
Ok, added it as a separate commit, but still within this PR (and sismplified per the next review comment)